eng/psakefile.ps1

```diff
@@ -75,6 +80,16 @@ Task pyqir-tests -Depends init {
     }
 }
 
+Task metawheel {
+    $wheelDir = Split-Path -Parent $wheelhouse
+    if (!(Test-Path $wheelDir)) {
+        New-Item -Path $wheelDir -ItemType Directory | Out-Null
+    }
+    Invoke-LoggedCommand {
+        & $python -m pip wheel --no-deps --wheel-dir $wheelDir $pyqir.meta.dir
```

Comment:
Ah, is `--wheel-dir` what keeps it from downloading dependencies of the metapackage? Makes sense!

Reply:
`--no-deps` skips local dep download